packages/insomnia-testing/src/generate/index.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+import { escapeJsStr, indent } from './util';
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+
+type Test = {
+  name: string,
+  code: string,
+};
+
+type Suite = {
+  name: string,
+  suites: Array<Suite>,
+  tests?: Array<Test>,
+};
+
+export async function generate(suites: Array<Suite>): Promise<string> {
```

Comment:
Don't think this needs to be `async`

```suggestion
export function generate(suites: Array<Suite>): string {
```